README.md

```diff
@@ -1,149 +1,194 @@
+Configuration files make use of two primary operators:
+
+* **Metavariables like $X, $LIST, or $METAVARIABLE.** Metavariable names can
+only contain uppercase characters. Metavariables are used to track a variable
+across a specific code scope.
+* **The '...' (splat) operator.** The splat operator abstracts away sequences
+so you don't have to sweat the details of a particular code pattern.
+
+Let's consider an example:
+
+```yaml
+rules:
+  - id: open-never-closed
+    patterns:
+      - pattern: $FD = open(...)
+      - pattern-not-inside: |
+          $FD = open(...)
+          ...
+          $FD.close()
+    message: "file object opened without corresponding close"
+    languages: [python]
+    severity: ERROR
+```
 
-#### Operators
+This rule looks for files that are opened but never closed. It accomplishes
+this by looking for the `open(...)` pattern _and not_ a following `close()`
+pattern. The `$FD` metavariable ensures that the same variable name is used
+in the `open` and `close` calls. The splat operator allows for any arguments
+to be passed to `open` and any sequence of code statements in-between the `open`
+and `close` calls. We don't care how `open` is called or what happens up to
+a `close` call, we just need to make sure `close` is called.
 
-`...` is the primary "match anything" operator. It can match sequences of characters, arguments, or statements. See [docs/config/simple.md](docs/config/simple.md) for more details!
+For a more complete introduction to the configuration format please see the
+[advanced configuration documentation](docs/config/advanced.md).
 
 #### Equivalences
 
-`sgrep` automatically searches for code that is semantically equivalent. For example, a pattern for
+Equivalences are another key concept in `sgrep`. `sgrep` automatically searches
+for code that is semantically equivalent. For example, the following patterns
+are semantically equivalent
 
-```text
+```python
 subprocess.Popen(...)
 ```
 
-will match
-
 ```python
-from subprocess import Popen as
- sub_popen
-result = sub_popen(“ls”)
+from subprocess import Popen as sub_popen
+result = sub_popen("ls")
 ```
 
-and other semantically equivalent configurations.
+For a full list of `sgrep` feature support by language see the
+[language matrix](docs/matrix.md).
+
+### Registry
 
-## Integrations
+As mentioned above, you may also specify a registry name as configuration.
+[r2c](https://r2c.dev) provides a [registry](https://github.com/returntocorp/sgrep-rules)
+of configuration files. These rules have been tuned on thousands of repositories
+using our [analysis platform](https://app.r2c.dev).
 
-See [integrations.md](docs/integrations.md)
+```
+$ docker run --rm -v "${PWD}:/home/repo" returntocorp/sgrep --config r2c
+```
 
-## Bug Reports
+## Resources
 
-Reports are welcome! Please open an issue on this project.
+* [r2c `sgrep` meetup slides](https://r2c.dev/sgrep-public2.pdf)
+* [Simple configuration documentation](docs/config/simple.md)
```

Comment:
I like the resources section but the simple + advanced conf are such important parts I feel they merit being mentioned earlier and with larger emphasis

Reply:
That makes sense to me. I'm happy to more prominently display them earlier. One question though: do you see value in keeping them separate? My thoughts were that the simple docs should be simple enough to be in the README, i.e. "Configuration" section above, then link out to advanced configuration. Does that make sense?

Reply:
Yes, I like that a lot!